Source/WebKit/UIProcess/Automation/BidiScriptAgent.cpp

```diff
@@ -59,7 +59,10 @@ void BidiScriptAgent::callFunction(const String& functionDeclaration, bool await
     // FIXME: handle non-BrowsingContext obtained from `Target`.
     std::optional<BrowsingContext> browsingContext = target->getString("context"_s);
     ASYNC_FAIL_WITH_PREDEFINED_ERROR_IF(!browsingContext, InvalidParameter);
-    ASYNC_FAIL_WITH_PREDEFINED_ERROR_IF(!session->webPageProxyForHandle(*browsingContext), WindowNotFound);
+
+    auto contextHandlePair = session->extractBrowsingContextHandles(*browsingContext);
+    ASYNC_FAIL_IF_UNEXPECTED(contextHandlePair);
```

Comment:
Nit: I would prefer to name the argument in the macro name, i.e., ASYNC_FAIL_IF_UNEXPECTED_RESULT or ASYNC_FAIL_IF_UNEXPECTED_VALUE.

That way it will read better. As-is, it reads as though "async fail if there is an unexpected context handle pair"